mysql-test/mysql-test-run.pl

```diff
@@ -5531,31 +5567,13 @@ ($)
   # start call.
   foreach my $server (all_servers()) {
     $server->{START}->($server, $tinfo) if $server->{START};
-    # If wsrep is on, we need to wait until the first
```

Comment:
This whole `start_servers()` sub should be restored to match 10.4 branch in verbatim.

Reply:
Done, except for $tinfo passed to mysql_server_wait